wayland/getting-started.txt

```diff
@@ -0,0 +1,73 @@
+Wayland
+________________________________________________________________________________
+
+Wayland [0] is a display protocol that aims to be a simpler and more modern
+replacement for the X Window System. The Wayland protocol follows a
+client–server model in which clients are the graphical applications requesting
+the display of pixel buffers on the screen, and the server (compositor) is the
+service provider controlling the display of these buffers.
+
+Wayland provides useful improvements over X11, including better multimonitor and
+HiDPI support [1], no screen tearing [2], and hardware accelerated video [3].
+
+Wayland has been in development since September of 2008 and is usable today for
+a large number of use-cases. Some hardware configurations (namely NVIDIA GPUs)
+[4] are poorly supported, though this is changing! See [5]. However, Wayland is
+seeing increased adoption in much larger projects, including GNOME [6] and KDE
+[7]. There is also work to make Firefox run on Wayland-only systems [8], and
+Chromium can already be fully built without Xorg libraries using Ozone [9]. If
+all you need is a web browser, a terminal, and dmenu, Wayland can work quite
+well for you.
+
+Because KISS does not support Wayland, users will have to use others'
+repositories or maintain their own. Luckily, many of the dependencies for
+Wayland are packaged in the main repositories, and only require some minimal
+tweaking in the build scripts to add Wayland support.
+
+Several user repositories already exist which provide Wayland packages,
+including $/Himmalerin/kiss-wayland and $/dilyn-corner/KISS-me
+
+Here is a list of packages you'll need when using Wayland:
+
++---------------+--------------------------------------------------------------+
+| Package       | Option                                                       |
++---------------+--------------------------------------------------------------+
+| mesa          | Add 'wayland' to -Dplatforms                                 |
+| gtk+3         | Add --enable-wayland=true                                    |
+| falkon        | Add -DNO_X11=TRUE                                            |
+| firefox       | Change --default-toolkit to cairo-gtk3-wayland               |
+| chromium      | Add ozone_platform_wayland=true                              |
+| xorg-server   | Add --enable-xwayland                                        |
+| libxkbcommon  | Add -Denable-wayland=true                                    |
```

Comment:
`libepoxy` is missing with `-Dx11=false \` .  Required for firefox.

Reply:
Again, forgotten :o